tensorboard/plugins/mesh_visualizer/metadata.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Internal information about the mesh plugin."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from tensorboard.compat.proto import summary_pb2
+from tensorboard.plugins.mesh_visualizer import plugin_data_pb2
+
+
+PLUGIN_NAME = 'mesh'
```

Comment:
Since the plugin name itself is just `mesh` I think it'd be a bit simpler to have the directory just be `tensorboard/plugins/mesh`.  The fact that it's a visualizer is already implicit in it being a tensorboard plugin.

The renaming would best be done at the end of review (just before merge) to avoid breaking review comment threads.

Reply:
sounds good